cif_core.dic

```diff
@@ -22076,6 +22140,117 @@ save_atom_site_aniso.matrix_b_su
 
 save_
 
+save_atom_site_aniso.matrix_beta
+
+    _definition.id                '_atom_site_aniso.matrix_beta'
+    _alias.definition_id          '_atom_site.tensor_beta'
+    _definition.update            2023-06-16
+    _description.text
+;
+    The symmetric anisotropic atomic displacement parameter (ADP) matrix \b,
+    which appears in a structure factor expression.
+
+    The contribution of the ADPs to the calculation of the structure factor is
+    given as:
+
+        T = exp(-1 * Sum(Sum(\b^ij^ * h~i~ * h~j~, j=1:3), i=1:3))
+
+    where \b^ij^ are the matrix elements, and h~m~ are the Miller indices.
+
+    The ADP matrix \b, is related to the ADP matrices U and B, as follows:
+
+                |a~1~*   0     0  |   |t^11^ t^12^ t^13^|   |a~1~*   0     0  |
+       \b = C * |  0   a~2~*   0  | * |t^12^ t^22^ t^23^| * |  0   a~2~*   0  |
+                |  0     0   a~3~*|   |t^13^ t^23^ t^33^|   |  0     0   a~3~*|
+
+    where C is a constant (2 * pi^2^ for U, and 0.25 for B), a~i~* is the
+    length of the respective reciprocal unit cell vector, and t represents
+    the individual anisotropic values, U^ij^ or B^ij^.
```

Comment:
Given that the Unicode branch is now merged in, we might want to update the text here as well to contain proper Unicode symbols.